salt/loader.py

```diff
@@ -2197,14 +2197,14 @@ def _process_virtual(self, mod, module_name, virtual_func="__virtual__"):
 
         return (True, module_name, None, virtual_aliases)
 
-    def run(self, method, *args, **kwargs):
+    def run(self, _func_or_method, *args, **kwargs):
```

Comment:
Why is this change necessary? I'm concerned we are changing the arg names on a publicly accessible method.

Reply:
These functions are the loader changes from @dwoz, unreleased.
They needed to be named more uniquely because if the target function also had a `method` argument, python would blow up.